### ISSUE TITLE
x64 shellcode support reverse_tcp_dns? 

### ISSUE THREAD

**Issue by kainpark7894** [rapid7/metasploit-framework#6858]
set payload windows/meterpreter/reverse_tcp_dns
(no problem)

set payload windows/x64/meterpreter/reverse_tcp_dns
(x64 No Supprot?)

set payload windows/x64/meterpreter/reverse_tcp

(LHOST:test.dns.net => ip change =>no dns information,The use of IP as before)

**Comment by egypt** [comment]
That is correct, there is no 64-bit `reverse_tcp_dns` stager. reverse_http(s) stagers do resolve on the victim, though.
